curl/deprecation.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+echo
+echo           \*\*\*\*\* DEPRECATION NOTICE \*\*\*\*\*
+echo
+echo This image is deprecated and will no longer be updated.
+echo This recent version of the image will continue to exist.
+echo
+echo For best support of \`curl\` please use one of the official
+echo \`curl\` images maintained by the curlimages community
+echo on Dockerhub.
+echo For details, visit https://hub.docker.com/r/curlimages/curl. Note that
+echo this image executes as special user \`curl_user\` and thus may not echo be
```

Comment:
There's an 'echo' out of place in this line

Reply:
thanks, fixed